lisa/operating_system.py

```diff
@@ -142,6 +145,7 @@ def _get_detect_string(cls, node: Any) -> Iterable[str]:
         cmd_result = typed_node.execute(cmd="cat /etc/os-release", no_error_log=True)
         yield get_matched_str(cmd_result.stdout, cls.__os_release_pattern_name)
         yield get_matched_str(cmd_result.stdout, cls.__os_release_pattern_id)
+        yield get_matched_str(cmd_result.stdout, cls.__os_release_pattern_idlike)
```

Comment:
Can it be recognized by ID, instead of ID_LIKE?

Reply:
If we don't add Class almalinux or add "almalinux" name_pattern under class RedHat, almalinux can't be recognized by ID.

Reply:
1. We should add name_pattern under class RedHat.
2. Can the almaLinux recognized by ID? What's value in ID and ID_LIKE?

Reply:
```
cat /etc/os-release
NAME="AlmaLinux"
VERSION="8.4 (Electric Cheetah)"
**ID="almalinux"
ID_LIKE="rhel centos fedora"**
VERSION_ID="8.4"
PLATFORM_ID="platform:el8"
PRETTY_NAME="AlmaLinux 8.4 (Electric Cheetah)"
ANSI_COLOR="0;34"
CPE_NAME="cpe:/o:almalinux:almalinux:8.4:GA"
HOME_URL="https://almalinux.org/"
DOCUMENTATION_URL="https://wiki.almalinux.org/"
BUG_REPORT_URL="https://bugs.almalinux.org/"

ALMALINUX_MANTISBT_PROJECT="AlmaLinux-8"
ALMALINUX_MANTISBT_PROJECT_VERSION="8.4"
```

Reply:
```
NAME="AlmaLinux"
VERSION="8.4 (Electric Cheetah)"
ID="almalinux"
ID_LIKE="rhel centos fedora"
```